docs/source/usingmame/defaultkeys.rst

```diff
@@ -1,106 +1,476 @@
+| **M**                              | Bet                                                                                                       |
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| **A**                              | High                                                                                                      |
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| **D**                              | Half Gamble                                                                                               |
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| **I**                              | Payout                                                                                                    |
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| **O**                              | Door                                                                                                      |
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| **Q**                              | Key In                                                                                                    |
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| **W**                              | Key Out                                                                                                   |
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| **S**                              | Low                                                                                                       |
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| **9**                              | Service                                                                                                   |
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| **0**                              | Book-Keeping                                                                                              |
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
```

Comment:
The poker and slots controls should be listed in this section as well.

Reply:
Added.